### PR TITLE
update CMake module to work with Connext 5.2

### PR DESCRIPTION
**Pull request by dirk-thomas**
This makes it build against Connext 5.2.

But currently the unit tests covering interoperability between different vendors fails with Connext 5.2.

### CODE REVIEW

File: connext_cmake_module/cmake/Modules/FindConnext.cmake [hunk 1]
```diff
@@ -213,8 +213,12 @@ if(NOT "${_NDDSHOME} " STREQUAL " ")
     set(Connext_DEFINITIONS "RTI_LINUX" "RTI_UNIX")
     if(EXISTS "${Connext_HOME}/bin/rtiddsgen2")
       set(Connext_DDSGEN2 "${Connext_HOME}/bin/rtiddsgen2")
-    else()
+    elseif(EXISTS "${Connext_HOME}/scripts/rtiddsgen2")
       set(Connext_DDSGEN2 "${Connext_HOME}/scripts/rtiddsgen2")
+    elseif(EXISTS "${Connext_HOME}/bin/rtiddsgen")
+      set(Connext_DDSGEN2 "${Connext_HOME}/bin/rtiddsgen")
+    else()
+      message(FATAL_ERROR "Could not executable 'rtiddsgen(2)'")
```
Comment (wjwwood) [inline connext_cmake_module/cmake/Modules/FindConnext.cmake]: Could not find?

**Comment by wjwwood** [comment]
+1

**Comment by dirk-thomas** [comment]
Please review.

**Comment by tfoote** [comment]
+1

**Pull request merged**